mlflow/server/js/package.json

```diff
@@ -59,13 +60,14 @@
     "eslint-plugin-promise": "3.6.0",
     "eslint-plugin-react": "7.4.0",
     "eslint-plugin-standard": "3.0.1",
-    "jest-localstorage-mock": "^2.3.0",
-    "react-app-rewired": "^1.5.2",
-    "react-app-rewire-define-plugin": "1.0.0"
+    "jest-localstorage-mock": "^2.4.0",
+    "react-app-rewire-define-plugin": "1.0.0",
+    "react-app-rewired": "^2.1.1",
+    "webpack": "^4.28.3"
   },
   "scripts": {
     "start": "react-app-rewired start",
-    "build": "react-app-rewired build",
+    "build": "react-app-rewired --max_old_space_size=4096 build",
     "test": "react-app-rewired test --env=jsdom",
```

Comment:
Should we do this? https://github.com/facebook/create-react-app/blob/master/CHANGELOG.md#the-default-jest-environment-was-changed-to-jsdom